### ISSUE TITLE
[0.22.0-rc][Android][Windows] Can't build release apk 

### ISSUE THREAD

**Issue by terrysahaidak** [facebook/react-native#6456]
Just tried to get signed apk, but build fails.
I'm just trying `./gradlew assembleRelease` and it fails.
Also, I've tried `react-native bundle --entry-file index.android.js --platform android --dev false --bundle-output android/app/src /main/assets/index.android.bundle` and it also fails with the same error (see below).
But when I use local cli 
`node node_modules/react-native/local-cli/cli.js bundle --entry-file index.android.js --platform android --dev false --bundle-output android/app/src /main/assets/index.android.bundle`
It was done without any errors.

Can I just use local-cli with gradlew?

I'm using: 
react-native-cli@0.1.10 (also, tried with 0.1.9)
react-native@0.22.0-rc


Bug log:
```
:app:bundleReleaseJsAndAssets                                                                                                                         
'which' is not recognized as an internal or external command,                                                                                         
operable program or batch file.                                                                                                                       
bundle: Created ReactPackager                                                                                                                         
                              uncaught error Error: NotFoundError: Cannot find entry file index.android.js in any of the roots: ["F:\\tasssik\\Works\\
GitHub\\gittermobile"]                                                                                                                                
    at DependencyGraph._getAbsolutePath (F:\tasssik\Works\GitHub\gittermobile\node_modules\node-haste\lib\index.js:278:13)                            
    at F:\tasssik\Works\GitHub\gittermobile\node_modules\node-haste\lib\index.js:215:30                                                               
    at tryCallOne (F:\tasssik\Works\GitHub\gittermobile\node_modules\promise\lib\core.js:37:12)                                                       
    at F:\tasssik\Works\GitHub\gittermobile\node_modules\promise\lib\core.js:123:15                                                                   
    at flush (F:\tasssik\Works\GitHub\gittermobile\node_modules\asap\raw.js:50:29)                                                                    
    at nextTickCallbackWith0Args (node.js:452:9)                                                                                                      
    at process._tickCallback (node.js:381:13)                                                                                                         
See logs C:\Users\s_tas\AppData\Local\Temp\react-packager.log                                                                                         
    at SocketClient._handleMessage (SocketClient.js:144:23)                                                                                           
    at BunserBuf.<anonymous> (SocketClient.js:53:42)                                                                                                  
    at emitOne (events.js:90:13)                                                                                                                      
    at BunserBuf.emit (events.js:182:7)                                                                                                               
    at BunserBuf.process (F:\tasssik\Works\GitHub\gittermobile\node_modules\bser\index.js:289:10)                                                     
    at F:\tasssik\Works\GitHub\gittermobile\node_modules\bser\index.js:244:12                                                                         
    at nextTickCallbackWith0Args (node.js:452:9)                                                                                                      
    at process._tickCallback (node.js:381:13)                                                                                                         
                                                                                                                                                     :
app:bundleReleaseJsAndAssets                                                                                                                         F
AILED                                                                                                                                                 
                                                                                                                                                      
FAILURE: Build failed with an exception.                                                                                                              
                                                                                                                                                      
* What went wrong:                                                                                                                                    
Execution failed for task ':app:bundleReleaseJsAndAssets'.                                                                                            
> Process 'command 'cmd'' finished with non-zero exit value 1                                                                                         
                                                                                                                                                      
```

**Comment by janicduplessis** [comment]
Windows support is broken in 0.22.0-rc, it is fixed on master at the moment and will be cherry picked in the 0.22 release.

**Comment by janicduplessis** [comment]
Not sure why it worked when you started the local-cli directly. Does it work if you just run the app in debug (react-native run-android)?

**Comment by terrysahaidak** [comment]
@janicduplessis yup, it work in debug.
maybe you know any hack to get it work before 0.22 will be released?

**Comment by terrysahaidak** [comment]
Currently fixed that by making changes as this commit https://github.com/facebook/react-native/commit/fd816b1349506fb40c4229e470c0ef5c32de1e80 does
Don't close until 0.22.0 will be shipped.

**Comment by janicduplessis** [comment]
Should be fixed in react-native@0.22.0-rc4

**Comment by browniefed** [comment]
@facebook-github-bot answered
